apps/astarte_realm_management_api/lib/astarte_realm_management_api_web/controllers/interface_controller.ex

```diff
@@ -24,6 +24,16 @@ defmodule Astarte.RealmManagement.APIWeb.InterfaceController do
 
   action_fallback Astarte.RealmManagement.APIWeb.FallbackController
 
+  def index(conn, %{"realm_name" => realm_name, "detailed" => "true"}) do
```

Comment:
The presence/absence of the `"detailed"` param could be checked inside the function rather than in the head, see e.g. what's being done on `create/2` and `async_operation` on lines 43-54

Reply:
Changed, so detailed parameter is checked inside the function `index`.